app/views/transcripts/charts/_bubble_courses.html.haml

```diff
@@ -10,6 +18,7 @@
 
       // Set chart options
       var options = {
+        tooltip: { isHtml: true }, //It will make the chart tooltip be html rendered instead of svg rendered
```

Comment:
May I ask you to put this option under the `titleTextStyle` key, for all the title setting be on the top. Just a opinion hahaha